code/modules/robotics/bot/medbot.dm

```diff
@@ -38,7 +44,29 @@
 	var/treatment_fire = "saline"
 	var/treatment_tox = "charcoal"
 	var/treatment_virus = "spaceacillin"
+	/// the stuff the bot injects when emagged
+	var/list/dangerous_stuff = list()
+	/// Set this to make the bot only inject all this crap
+	var/list/override_reagent = list()
+	/// They'll stop stop injecting that crap if the patient has the per-inject amount
+	var/override_reagent_limit_mult = 0.9
 	var/terrifying = 0 // for making the medbots all super fucked up
+	/// List of drugs that terrifying derelist bots will inject
+	var/static/list/terrifying_meds = list("formaldehyde" = 15,
+																				"ketamine" = 5,
+																				"pancuronium" = 5,
+																				"haloperidol" = 15,
+																				"morphine" = 20,
+																				"cold_medicine" = 40,
+																				"simethicone" = 10,
+																				"sulfonal" = 5, /* its an oldetimey sedative */
+																				"atropine" = 30,
+																				"methamphetamine" = 20,
+																				"ethanol" = 20, /* rubbing alcohol */
+																				"fog" = 20,
+																				"ether" = 10,
+																				"chlorine" = 10, /* disinfectant */
+																				"mercury" = 5)
```

Comment:
```suggestion
	var/static/list/terrifying_meds = list("formaldehyde" = 15,
																				"ketamine" = 25,
																				"pancuronium" = 5,
																				"haloperidol" = 15,
																				"morphine" = 20,
																				"cold_medicine" = 40,
																				"simethicone" = 10,
																				"sulfonal" = 5, /* its an oldetimey sedative */
																				"atropine" = 10,
																				"methamphetamine" = 30,
																				"ethanol" = 20, /* rubbing alcohol */
																				"ether" = 10,
																				"chlorine" = 10, /* disinfectant */
																				"mercury" = 5)
```

Reply:
Sure! Though, the reason for all the atropine was that it was an overdose, but I suppose that it'd still be pretty debilitating like that!